### ISSUE TITLE
How did you promote your first mainstream open source project ?

### ISSUE THREAD

**Issue by antony-jr** [SBoudrias/ama#26]
I really want to know this! :dog:

**Comment by SBoudrias** [comment]
Hey Antony,

I guess my first mainstream OSS project was [Inquirer](https://github.com/SBoudrias/Inquirer.js/)

So for me, what I did is first of all check with other projects that could benefit from Inquirer and send PR to integrate my library with theirs. That was probably my first big contribution to Yeoman. I think getting some amount of usage is pretty important before promoting elsewhere as no one really want to use an unproved/unused library.

Once it got a few users, I posted on [r/javascript](https://www.reddit.com/r/javascript/) where it was well received and I sent an email to the JS Weekly newsletter presenting my lib, and they decided to include it in their next newsletter.

I probably posted on my twitter too, but I don't think this help much unless you have lots of followers (which isn't my case - and was even less true when I got started with OSS)
